src/components/AssetsTable/index.jsx

```diff
@@ -429,6 +434,18 @@ export default class AssetsTable extends React.Component {
         currentAsset.external_url,
       );
 
+      currentAsset.display_name = (
+        <span data-identifier="asset-file-name">{currentAsset.display_name}</span>
+      );
+
+      currentAsset.content_type = (
+        <span data-identifier="asset-content-type">{currentAsset.display_name}</span>
```

Comment:
Is this also supposed to be `currentAsset.display_name`?

Reply:
It's supposed to be content_type. I copy-pasted. Thanks for catching that.